packages/jsapi-shim/src/dh.types.ts

```diff
@@ -865,7 +865,17 @@ export interface TableMap extends Evented {
   getTable(key: object): Promise<Table>;
 }
 
+export interface WorkerHeapInfo {
+  readonly maximumHeapSize: number;
+  readonly freeMemory: number;
+  readonly totalHeapSize: number;
+  getMaximumHeapSize: () => number;
+  getFreeMemory: () => number;
+  getTotalHeapSize: () => number;
```

Comment:
Looking at the API code, these getters don't exist in this format. GWT will convert the Java into a JS getter since it has `@JSProperty` decorator

So it's just the readonly values. If you were to log the result of `getWorkerHeapInfo()` then it should show you the response which I think will just have the 3 properties